NEWS

```diff
@@ -80,6 +82,15 @@ NEWS - user visible changes				-*- outline -*-
 ** new compiler command line option to list the known runtime exception names
    and fatality `cobc --list-exceptions`
 
+** New -std options:
+
+   gcos            GCOS compatibility
+   gcos-strict     GCOS compatibility                - strict
+
+   We define the GCOS dialect based on the COBOL85 standard, with new
+   dialect configuration options accompanying each specificity
+   introduced by the dialect
```

Comment:
minor typo: either end with a full-stop or start lower case, otherwise this file is "done"